lib/modules/manager/composer/schema.ts

```diff
@@ -3,6 +3,7 @@ import { logger } from '../../../logger';
 import { readLocalFile } from '../../../util/fs';
 import { regEx } from '../../../util/regex';
 import { Json, LooseArray, LooseRecord } from '../../../util/schema-utils';
+import {BitbucketTagsDatasource} from "../../datasource/bitbucket-tags";
 import { GitTagsDatasource } from '../../datasource/git-tags';
```

Comment:
Do you _really_ want to have `feat(manager/composer): Use bitbucket-tags for vcs repos if it has bitbucket.org URL (#24065)` as the PR title?

I assume the `(#24065)` part is a typo?

Reply:
> Why do you assume it is a typo?

We usually don't link to an issue or discussion within the PR title.

Closing an issue with the keywords: `Closes #24065` in the PR body is enough. But `#24065` is a Discussion, so it won't be automatically closed by GitHub after the PR is merged.

Reply:
Ok, thank you for that information. I have removed the discussion number from the PR title.